Handle null values in k3d.configureComponents (#128)
cloudogu/ces-build-lib: CHANGELOG.md

## [Unreleased]
### Changed

> - change `k3d.configureComponents` so that `null` values will delete the components from the config.

Reviewer: This sentence is a bit misleading because it references unnamed components. How about this?

```suggestion
- change `k3d.configureComponents` so that `null` values disable single components so they won't be installed.
```
